http-protocol-binding.md

```diff
@@ -429,6 +429,83 @@ Content-Length: nnnn
 
 ```
 
+### 3.4. Multipart Structured Content Mode
```

Comment:
WDYT about adding support for implicit values? Something like Elasticsearch's Bulk API, where some fields can be omitted if they share common value:
https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html#docs-bulk-api-desc

this way it can help reducing the traffic when big amounts of events (e.g. with the same specversion) need to be transmitted.

Reply:
This looks like it fits better the `binary` mode